### PR TITLE
GSREN3D-425 -  graph result

### PR DESCRIPTION
**Pull request by toni-garcia-camptocamp**
GSREN3D-425 -  graph result

### JIRA issue

https://jira.camptocamp.com/browse/GSREN3D-425

FIRST MERGE THIS PR IN BACK : https://github.com/sigrennesmetropole/cooperation_jn_backend/pull/10

### Description
image
image

### CODE REVIEW

File: src/components/results/AutocalsolResultGraph.vue [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+      {
+        name: 'Intersection',
+        data: intersectionData,
+        color: '#10B981',
+        lineWidth: 0,
+        fillOpacity: 1,
+      },
+    ],
+    credits: {
+      enabled: false, // Disable the Highcharts.com credit link
```
Comment (ismailsunni) [inline src/components/results/AutocalsolResultGraph.vue:122]: do we need to buy the license? Or is there a free tier for our use case?

**Review by ismailsunni — COMMENTED** [review]
The code is LGTM. Two minor comments.

**Comment by toni-garcia-camptocamp** [comment]
> Nice 👍 What is "stuffingData" for?

/**
 * By default, we have one value for one hour. When we calculate the intersection, the precision of the intersection graph is not very good
 * So we need to add dummy data to have a more accurate intersection graph
 *
 * Example: 10h we have a production of 1000W and 11h we have a production of 2000W
 * The intersection graph will be a straight line between 1000W and 2000W
 * With the stuffing, we will have 10 values between 1000W and 2000W : 1000W, 1100W, 1200W, 1300W, 1400W, 1500W, 1600W, 1700W, 1800W, 1900W, 2000W
 */

I have one comment to explain stuffingData

File: src/assets/icons/eclipse-full.svg
[the remark this answers is not in the document]
Comment (toni-garcia-camptocamp) [reply src/assets/icons/eclipse-full.svg:1]: You right, this svg is use nowhere
I have delete it

File: src/components/results/AutocalsolResultGraph.vue (hunk 1 above)
Comment (toni-garcia-camptocamp) [reply src/components/results/AutocalsolResultGraph.vue]: According to the highchart website (https://www.highcharts.com/blog/download/) we can use highchart for free if Rennes is : "a non-profit organization, personal website, or school project ". But we have to display the credit 

image


I have change this line to display the credit

**Pull request merged**